towhee/__init__.py

```diff
@@ -44,55 +45,86 @@ class _PipelineWrapper:
         pipeline (`towhee.Pipeline`):
             Base `Pipeline` instance for which this object will provide a wrapper for.
     """
+
     def __init__(self, pipeline_: Pipeline):
         self._pipeline = pipeline_
 
-    def __call__(self, *args) -> Tuple[Any]:
+    def __call__(self, *args) -> List[Tuple]:
         """
         Wraps the input arguments around a `Dataframe` for Pipeline.__call__(). For
         example:
         ```
         >>> p = pipeline('some-pipeline')
-        >>> single_result = p(arg0, arg1)
-        >>> multi_result = p([(arg0_0, arg1_0), (arg0_1, arg1_1)])
+        >>> result = p(arg0, arg1)
         ```
-
         """
-
         if not args:
-            return []
+            raise RuntimeError('Input data is empty')
 
-        # Support both single-input and multi-input (via list).
-        if len(args) == 1 and isinstance(args[0], list):
-            inputs = args[0]
-        else:
-            inputs = [args]
+        vargs = []
+        for arg in args:
+            vtype = type(arg).__name__
+            vargs.append(Variable(vtype, arg))
+        vargs = tuple(vargs)
 
+        # Process the data through the pipeline.
         in_df = DataFrame('_in_df')
-        for tup in inputs:
```

Comment:
Are we purposely removing the ability to deal with multiple inputs?

Reply:
Yeah, I think `pipeline()` is a very simple interface, users can send one input data, get the result if run success else raises an exception if have multiple inputs, users can write a loop and call `pipeline()` continually.

Reply:
We will support `data_source` op, such as `local_data_source, `hdfs_data_source`, and so on, these can provide continuous input data.